tests/Zend/Db/Statement/TestCommon.php

```diff
@@ -329,6 +329,10 @@ public function testStatementSetFetchModeInvalidException()
             $this->assertTrue($e instanceof Zend_Db_Statement_Exception,
                 'Expecting object of type Zend_Db_Statement_Exception, got '.get_class($e));
             $this->assertRegExp('#invalid fetch mode#i', $e->getMessage());
+        } catch (Throwable $e) {
+            $this->assertTrue($e instanceof ValueError,
+                'Expecting object of type Zend_Db_Statement_Exception, got '.get_class($e));
+            $this->assertRegExp('#must be a bitmask#i', $e->getMessage());
```

Comment:
This and other Throwable, ValueError fixes to tests are wrong in this PR. Code must be adjusted to keep previous behavior, even if it's wrong behavior.

For example, outer code expects Zend_Db_Statement_Exception to be thrown, but instead, ValueError is thrown, and the whole application breaks. Even old PHPUnit breaks because \Error is not a subclass of \Exception.

My version of the problem fix:
- https://github.com/zf1s/zf1/pull/85